pages/api/me/password.js

```diff
@@ -55,23 +48,21 @@ async function put(req, res) {
       return;
     } 
 
-    const session = await Session.getSession(session_token);
-    const forcePasswordChange = await People.updatePassword({id: session.people_id, oldPassword: currentPassword, newPassword});
-  
-    req.session.cache = {
-      ...req.session.cache,
-      forcePasswordChange,
-    };
-    await req.session.save();
+    const { sessionToken } = req.user;
```

Comment:
We may need to do a fallback mechanism like so:

```suggestion
    const { sessionToken } = req.session ?? req.user;
```